vmm/src/lib.rs

```diff
@@ -237,9 +254,19 @@ pub fn start_vmm_thread(
     api_event: EventFd,
     api_sender: Sender<ApiRequest>,
     api_receiver: Receiver<ApiRequest>,
+    debug_path: Option<String>,
+    debug_event: EventFd,
+    vm_debug_event: EventFd,
```

Comment:
These two `EventFd` should be behind the `gdb` feature. We don't want to create these extra eventfds when it's not needed.